### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by mraible**
## Description:

- Change `code` link to `blog` in header (requested by @lindsaybrunner-okta).
- Add GitHub project link to sign-in widget (discussed with @lboyette-okta).
- Made header non-static when viewing blog posts. 

Screenshot of first two changes:

github-link

The 3rd change allows us to deep-link into blog posts. Currently, when you try to deep-link, it hides the section you're linking to.

header-hides

After this change, you'll see the section you're linking to:

header-scrolls

## Type of Pull Request:
- [ ] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [x] Functional (CSS changes, JS updates, or layout modifications)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [x] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:

@okta/devex-sample-contributors

### CODE REVIEW

**Comment by bretterer** [comment]
I am not a fan of making the header non-sticky for deep linking.  A solution to fix this is the following (from our old blog)

```
.post a.heading-link {
  color: #999999;
  text-decoration: none;
  cursor: pointer;
  visibility: hidden;

}

.post a.heading-link:hover {
  color: #1a32bd;
}

.post h2:hover a.heading-link {
  visibility: visible;
}

.post h2 .glyphicon-link {
  font-size: 14px;
  margin-left: 6px;
  cursor: pointer;
  top: -1px;
}

.post h2:before {
  display: block;
  content: " ";
  margin-top: -90px;
  height: 90px;
  visibility: hidden;
}

@media (min-width: 768px) {
  .post h2:before {
    display: block;
    content: " ";
    margin-top: -75px;
    height: 75px;
    visibility: hidden;
  }
}
```

The important part is the `.post h2:before` section which adds an invisible margin to the top of the heading that lets it work with the sticky header.

**Comment by mystiberry-okta** [comment]
Has anyone looked at the API doc to make sure there's no ill effects? And the landing page?

**Comment by rchild-okta** [comment]
Is there a reason why we're replacing the code link, rather than adding another menu item for blog? Along the same lines, now that we're removing it, how will developers navigate to the code samples?

**Comment by mraible** [comment]
@bretterer I'm not a fan of sticky headers in general, but I'll try your approach.

@rchild-okta I'll revert this change and simply add "Blog" as another link after code, per Tom's suggestion.

**Comment by mraible** [comment]
@bretterer I tried adding the following, but it doesn't seem to work:

```css
  #blog-post .post-block:before {
    display: block;
    content: " ";
    margin-top: -90px;
    height: 90px;
    visibility: hidden;
  }
```

Any ideas how to make the following link show the proper header and not hide it?

https://developer.okta.com/blog/2017/04/26/bootiful-development-with-spring-boot-and-angular#configure-cors-for-spring-boot

Maybe it needs to be done for all h2s and h3s?

**Comment by mraible** [comment]
FWIW, I added the link to "Code" back in and reverted my change for the scrolling header.

**Comment by mystiberry-okta** [comment]
@bretterer or @mraible is this ready to merge?

**Comment by mraible** [comment]
@mystiberry-okta I fixed some things so yes, it is ready IMO. Currently, it only adds a "Blog" link in the header and added a "GitHub Project" link to the TOC for the Sign-In Widget.

**Comment by mystiberry-okta** [comment]
@bretterer can you rereview?

File: 3rd_party_notices/Android_Mobile.html [hunk 1]
```diff
@@ -72,6 +72,9 @@ <h1 class="logo"><a href="/">Okta</a></h1><!-- START Primary Nav -->
               <li>
                 <a href="/code/">Code</a>
               </li>
+              <li>
+                <a href="/blog/">Blog</a>
```
Comment (jmelberg-okta) [inline 3rd_party_notices/Android_Mobile.html]: I'm a huge fan of having this in the main nav as shown. However, we should remove the link in the footer to remove duplication.

Comment (mraible) [inline 3rd_party_notices/Android_Mobile.html]: Done.

**Comment by mraible** [comment]
@bretterer @jmelberg-okta I've removed the "Blog" link from the footer, added it in the header, and changed the Twitter handle to OktaDev. Those are the only changes in this PR.

**Comment by jmelberg-okta** [comment]
Thanks @mraible! Can you rebase off master to clean up this PR? Then we should be able to get this merged in. 👍

**Comment by mraible** [comment]
@jmelberg-okta I merged from `master`, so there's no longer conflicts.

**Comment by mraible** [comment]
Closing because of too many commits. Deleting my fork; will create a new PR.

**Pull request closed without merging**